src/axom/inlet/examples/example1_expected_documentation.rst

```diff
@@ -118,7 +118,7 @@ Description: This is the solver sub-table in the thermal_solver table
    * - rel_tol
      - description for solver rel tol
      - 
-     - 
+     - 0.500000 to 100.700000

```

Comment:
The title of this column needs to be more general... its not just "Range" anymore. "Valid Values/Range"